pkg/lightning/restore/restore_test.go

```diff
@@ -815,21 +819,22 @@ func (s *tableRestoreSuite) TestImportKVFailure(c *C) {
 func (s *tableRestoreSuite) TestCheckRequirements(c *C) {
 	controller := gomock.NewController(c)
 	defer controller.Finish()
-	mockBackend := mock.NewMockBackend(controller)
-	backend := backend.MakeBackend(mockBackend)
-
-	ctx := context.Background()
-
-	mockBackend.EXPECT().
-		CheckRequirements(ctx, gomock.Any()).
-		Return(errors.Annotate(context.Canceled, "fake check requirement error"))
-	rc := &Controller{
-		cfg:     &config.Config{App: config.Lightning{CheckRequirements: true}},
-		backend: backend,
-	}
-
-	err := rc.checkRequirements(ctx)
-	c.Assert(err, ErrorMatches, "fake check requirement error.*")
+	// mockBackend := mock.NewMockBackend(controller)
```

Comment:
Please fix this test or remove it if we don't need it anymore

Reply:
ok, I'll remove it.